Makefile

```diff
@@ -10,6 +10,9 @@ BUNDLE_CHANNELS := --channels=$(CHANNELS)
 DEFAULT_CHANNEL=7.x
 BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+# Container runtime engine used for building the images
+BUILDER ?= podman
```

Comment:
So we can now use podman with cekit? 😁

Reply:
for operator  as it does not have behave tests, I would say yes.